src/feature/followUnfollow/__tests__/followUnfollowAction.test.js

```diff
@@ -0,0 +1,190 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import axios from 'axios';
+import {
+  getFollowing, followAuthor, unfollowAuthor, clearFollowing
+} from '../followUnfollowAction';
+import LocalStorage from '../../../__mocks__/localStorage';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+const store = mockStore({});
+
+let storage;
+const token = 'Invalid token';
+describe('Follow Author Tests', () => {
+  beforeEach(() => {
+    moxios.install(axios);
+    storage = window.localStorage.setItem('token', token);
+    window.localStorage = new LocalStorage();
+  });
+  afterEach(() => {
+    moxios.uninstall(axios);
+    store.clearActions();
+    window.localStorage = storage;
+  });
+  test('Follow Author Tests', () => {
```

Comment:
@raymond42 Aren't we supposed to use 'describe' and 'it' blocks? Kindly update from test to it